frontend/src/Frontend/UI/Wallet.hs

```diff
@@ -188,8 +188,7 @@ uiKeyItem model i d = do
             td $ dynText $ _account_notes <$> account
             td $ dynText $ ffor account $ \a -> case _account_balance a of
               Nothing -> "Unknown"
-              Just b -> tshow (unAccountBalance b) <> " KDA"
-
+              Just b -> tshow (unAccountBalance b) <> " KDA" <> maybe "" (const "*") (_account_unfinishedCrossChainTransfer a)
```

Comment:
Probably should have a tooltip showing what this means. Should run this by Kadena anyway.

Reply:
Yeah, will cover this in the demo.